DisaggregateFileRecord.py

```diff
@@ -0,0 +1,172 @@
+# Initial revision 2011 Sept 13 by lmz
```

Comment:
Why do we need this in Wrangler??  This is info specific to SF-CHAMP so should be kept there.  Keeping it both places will for sure make us get out of sync somewhere and accidentally use wrong one which could have disastrous consequences :-(